neuvector-dbgen.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: neuvector-dbgen
+  version: 0_git20240423
+  epoch: 0
+  description: NeuVector vulnerability database generator for the SUSE NeuVector Container Security Platform
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 8f334ddf3b99aefeef7884e4f126c03e7f5a44fa288b9aaba902b410150d27c9
+      uri: https://github.com/neuvector/vul-dbgen/archive/23c9cea6be0762c825673803624300cc4a720d19.tar.gz
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: .
+      output: dbgen
+      prefix: usr/local
+      vendor: true
+
+  - uses: strip
+
+subpackages:
+  - name: neuvector-db-updater
+    description: NeuVector DB Updater
+    dependencies:
+      runtime:
+        - ca-certificates
+        - curl
+    pipeline:
+      - uses: go/build
+        with:
+          modroot: updater
+          packages: .
+          output: updater
+          prefix: usr/local
+          subpackage: true
+          vendor: true
+      - uses: strip
+
+test:
+  pipeline:
+    - runs: |
+        dbgen --help
```

Comment:
Looks like this --help is exiting non-zero :(

Reply:
How on earth....

Reply:
Updated the test to just check for the executable. While --help is built in, it still returns nonzero as if it wasn't 🤦. This is great